@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHeadingHold.sqf

```diff
@@ -0,0 +1,72 @@
+params ["_heli", "_deltaTime"];
+#include "\fza_ah64_sfmplus\headers\core.hpp"
+
+private _pidHdg     = _heli getVariable "fza_sfmplus_pid_hdgHold";
+private _pidTrn     = _heli getVariable "fza_sfmplus_pid_trnCoord";
+//_pidTrn set ["kp", KP];
+//_pidTrn set ["ki", KI];
+//_pidTrn set ["kd", KD];
+private _curHdg     = getDir _heli;
+private _desiredHdg = _heli getVariable "fza_ah64_hdgHoldDesiredHdg";
+private _hdgError   = [_curHdg - _desiredHdg] call CBA_fnc_simplifyAngle180;
+private _curVel     = vectorMagnitude [velocityModelSpace _heli # 0, velocityModelSpace _heli # 1];
+private _subMode    = _heli getVariable "fza_ah64_hdgHoldSubMode";
+private _attSubMode = _heli getVariable "fza_ah64_attHoldSubMode";
+private _output     = 0.0;
+//Breakout values expand as the aircraft goes faster to provide good pedal response
+//at a hover. The expanded range is meant to de-sensitize the pedasl in order to 
+//prevent disengaging the heading hold mode during cruise flight
+private _breakoutValue = 0.0;
+if (_attSubMode == "pos") then {
+    _breakoutValue = HDG_HOLD_BREAKOUT_VALUE;
+};
+if (_attSubMode == "vel") then {
+    _breakoutValue = VEL_HOLD_BREAKOUT_VALUE;
+};
+if (_attSubMode == "att" || _curVel > HDG_HOLD_SPEED_SWITCH_ACCEL) then {
+    _breakoutValue = ATT_HOLD_BREAKOUT_VALUE;
+};
+//If we are on the ground, or if the force trim is interupted, or the pilot has exceeded
+//the breakout values for the pedals, then heading hold is not active (doing work)
+//otherwise, heading hold is ALWAYS active
+if (isTouchingGround _heli
+    || _heli getVariable "fza_ah64_forceTrimInterupted" 
+    || fza_sfmplus_pedalLeftRight <= -_breakoutValue 
```

Comment:
Could this be simplified to `abs fza_sfmplus_pedalLeftRight <= _breakoutValue`

Reply:
Pedals are a -1 to 1 value not 0 to 1 value